### PR TITLE
doc: Add a note to the SIM activation instructions

### PR DESCRIPTION
**Pull request by divipillai**
Add a note about accepting either EID or ICCID
to the SIM activation instructions in
Thingy:91 and nRF9160 user guides.

Also, added include (rst syntax) where the steps are same in 
Thingy:91 and nR9160 user guides.

Signed-off-by: divya pillai<divya.pillai@nordicsemi.no>

### CODE REVIEW

**Comment by divipillai** [comment]
The doc changes were reviewed and approved by Cole hafner (through teams).

**Review by jayteemo — APPROVED** [review]
approved, but please fix the `password your chose` wording

File: doc/nrf/ug_nrf9160_gs.rst [hunk 1]
```diff
@@ -217,23 +222,28 @@ To create an nRF Cloud account, complete the following steps:
    If you closed the registration dialog box, you can repeat Step 1 and then click :guilabel:`Already have a code?`.
    Then enter your email and the verification code.
 
-You can now log in with your email and the password you chose.
+You can now log in to `nRF Cloud`_ with your email and password.
 After logging in, you are taken to the dashboard view that displays your device count and service usage.
+Next, you need to activate the SIM card for the |DK|.
```
Comment (wiba-nordic) [inline doc/nrf/ug_nrf9160_gs.rst:227]:
```suggestion
Next, you need to activate the SIM card you will use in the |DK|.
```
Current version can be interpreted like the SIM card needs to be activated specifically for the DK, which is not the case.

File: doc/nrf/ug_nrf9160_gs.rst [hunk 2]
```diff
@@ -192,20 +192,25 @@ If you experience any problems during the process, restart the Programmer app by
 
 .. _nrf9160_gs_connecting_dk_to_cloud:
 
-Connecting the DK to nRF Cloud
-******************************
+.. |DK| replace:: nRF9160 DK
```
Comment (wiba-nordic) [inline doc/nrf/ug_nrf9160_gs.rst:195]: You need to switch the order of the replace definition and the reference tag, it's currently breaking the build.

**Review by wiba-nordic — CHANGES_REQUESTED** [review]

**Review by wiba-nordic — APPROVED** [review]

**Comment by umapraseeda** [comment]
We could also add the 'Add LTE Device' screenshot (available in the DK guide) to the Thingy:91 guide too (before the explanations of Device ID and Ownership code).

**Comment by wiba-nordic** [comment]
On that note, here's a more recent version of that screenshot. There's small changes that have been made, it seems.
nrfcloud_add_lte_device

**Comment by divipillai** [comment]
> On that note, here's a more recent version of that screenshot. There's small changes that have been made, it seems. nrfcloud_add_lte_device

I can add this image to both DK and thingy:91 user guides.

**Review by umapraseeda — APPROVED** [review]
LGTM!

**Comment by divipillai** [comment]
@jhn-nordic : Added you as a reviewer to review the note ([doc/nrf/ug_thingy91_gsg.rst](https://github.com/nrfconnect/sdk-nrf/pull/8318/files#diff-90d0e4bb792f25c3e98c1eeb33e528b8d7ff1ea2413b4975771ffeb64ec4fcd0)) I have updated in the Thingy:91 user guide's [Update the modem firmware on the nRF9160 SiP](http://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-8137/nrf/ug_thingy91_gsg.html#id40)[](http://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-8137/nrf/ug_thingy91_gsg.html#update-the-modem-firmware-on-the-nrf9160-sip) section. 
As discussed, not adding this note to the "through USB" section because the flow differs when using USB or debug probe.

**Review by jhn-nordic — APPROVED** [review]

**Pull request merged**